libs/utils/android/workloads/uibench.py

```diff
@@ -162,4 +178,15 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         System.set_airplane_mode(self._target, on=False)
         Screen.set_brightness(self._target, auto=True)
 
+    def _perform_action(self, action, delay_s=0):
```

Comment:
This way to perform actions is currently "functionally broken" because it generates tasks on the target side which can affect the test behaviours.

More details in #329, we should fix the System API before adding interactivity features to existing tests.